doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,36 @@
 
 <h3>New features since last release</h3>
 
+* A new function called :func:~.to_openqasm has been added, which allows for converting PennyLane circuits to OpenQASM 2.0 programs.
```

Comment:
```suggestion
* A new function called :func:`~.to_openqasm` has been added, which allows for converting PennyLane circuits to OpenQASM 2.0 programs.
```
Guess this might help

Reply:
Thank you @JerryChen97!